class TestExceptions(object):
    def test_exceptions_http_error(self):
        assert 5 == 5
